projects/storefrontstyles/scss/components/organization/_organization-home.scss

```diff
@@ -0,0 +1,55 @@
+.CompanyPageTemplate {
```

Comment:
I like that this styling is contained for the my company home page.
It would be nice as an improvement if we could figure out a way for the styles to be shared with other home pages ( like my account).